app/src/main/kotlin/com/softteco/template/MainActivity.kt

```diff
@@ -34,16 +36,17 @@ class MainActivity : ComponentActivity() {
     @Named("themeMode")
```

Comment:
Could we remove "themeMode" and use one DataStore for preferences?

Reply:
If we remove it from DataStore module, this  `@Named("themeMode")` will be not needed.